scripts/pi-hole/js/debug.js

```diff
@@ -0,0 +1,65 @@
+// Credit: http://stackoverflow.com/a/10642418/2087442
+function httpGet(ta,theUrl)
+{
+    var xmlhttp;
+    if (window.XMLHttpRequest)
+    {
+    // code for IE7+
+        xmlhttp = new XMLHttpRequest();
+    }
+    else
+    {
+    // code for IE6, IE5
+        xmlhttp = new ActiveXObject("Microsoft.XMLHTTP");
+    }
+    xmlhttp.onreadystatechange=function()
+    {
+        if (xmlhttp.readyState === 4 && xmlhttp.status === 200)
+        {
+            ta.show();
+            ta.empty();
+            ta.append(xmlhttp.responseText);
+        }
+    };
+    xmlhttp.open("GET", theUrl, false);
+    xmlhttp.send();
+}
+
+function eventsource() {
+    var ta = $("#output");
+    var upload = $( "#upload" );
+    var checked = "";
+
+    if(upload.prop("checked"))
+    {
+      checked = "upload";
+    }
+
+    // IE does not support EventSource - load whole content at once
+    if (typeof EventSource !== "function") {
+        httpGet(ta,"/admin/scripts/pi-hole/php/debug.php?IE&"+checked);
+        return;
+    }
+
+    var host = window.location.host;
+    var source = new EventSource("/admin/scripts/pi-hole/php/debug.php?"+checked);
+
+    // Reset and show field
+    ta.empty();
+    ta.show();
+
+    source.addEventListener("message", function(e) {
+        ta.append(e.data);
+    }, false);
+
+    // Will be called when script has finished
+    source.addEventListener("error", function(e) {
+        source.close();
```

Comment:
I'd like to keep that as it is. Force the users to at least reload the page before sending another debug.